src/components/base-control.js

```diff
@@ -33,21 +33,25 @@ const propTypes = {
   // Stop map click
   captureClick: PropTypes.bool,
   // Stop map double click
-  captureDoubleClick: PropTypes.bool
+  captureDoubleClick: PropTypes.bool,
+  // Stop map pointer move
+  capturePointerMove: PropTypes.bool
 };
 
 const defaultProps = {
   captureScroll: false,
   captureDrag: true,
   captureClick: true,
-  captureDoubleClick: true
+  captureDoubleClick: true,
+  capturePointerMove: true
```

Comment:
Let's set the default to `false` to avoid breaking existing custom controls.

Reply:
Ok that's done